### PR TITLE
Error cleanup

### PR DESCRIPTION
**Pull request by raxo2222**

### CODE REVIEW

File: localization/english/CWP/CWP_buildings_l_english.yml [hunk 1]
```diff
@@ -120,25 +120,53 @@
...
   bg_military_research: "Military Research Institutions"
-  bg_nuclear_power: "Nuclear power industry"
+  bg_nuclear_power: "Nuclear Power Industries"
   bg_nuclear_research: "Nuclear Research Institutions"
   bg_phosphorus_mining: "Phosphorus Mines"
   bg_rare_earth_elements_mining: "Rare Earths Mines"
   bg_technology: "Technological Institutions"
-  bg_telecommunications: "Telecommunications industry"
-  bg_tourism: "Tourism industry"
+  bg_telecommunications: "Telecommunications Industries"
```
Comment (DZappala) [inline localization/english/CWP/CWP_buildings_l_english.yml:144]: syntax error: double plural.
```suggestion
  bg_telecommunications: "Telecommunication Industries"
```

File: localization/english/CWP/CWP_buildings_l_english.yml [hunk 2]
```diff
@@ -120,25 +120,53 @@
...
+  bg_subsistence_agriculture: "Subsistence Agriculture"
+  bg_oil_rig: "Oil Rigs"
+  bg_natural_gas_well: "Natural Gas Wells"
+  bg_automobile_manufactories: "Automobile Manufactories"
+  bg_motor_industry: "Motor Industries"
+  bg_power: "Power Plants"
+  bg_combustion_power: "Fossil Fuel Plants"
+  bg_renewable_power: "Renewable Plants"
+  bg_geothermal_power: "Geothermal Plants"
+  bg_tidal_power: "Todal Plants"
```
Comment (DZappala) [inline localization/english/CWP/CWP_buildings_l_english.yml:157]:
```suggestion
  bg_tidal_power: "Tidal Plants"
```
should be

File: localization/english/CWP/CWP_buildings_l_english.yml [hunk 3]
```diff
@@ -120,25 +120,53 @@
...
+  bg_biomass_power: "Biomass Plants"
+  bg_electronics: "Electronics Industries"
+  bg_research_institutes: "Research Institutions"
+  bg_military_headquarters: "Military Headquarters"
+  bg_civil_defense: "Civil Defense"
```
Comment (DZappala) [inline localization/english/CWP/CWP_buildings_l_english.yml:162]:
```suggestion
  bg_civil_defense: "Civil Defense Sectors"
```

File: localization/english/CWP/CWP_buildings_l_english.yml [hunk 4]
```diff
@@ -120,25 +120,53 @@
...
+  bg_foreign_service: "Foreign Service"
```
Comment (DZappala) [inline localization/english/CWP/CWP_buildings_l_english.yml:163]:
```suggestion
  bg_foreign_service: "Foreign Service Sectors"
```

**Review by DZappala — CHANGES_REQUESTED** [review]
Minor changes

File: localization/english/CWP/CWP_buildings_l_english.yml (hunk 1 above)
Comment (raxo2222) [reply localization/english/CWP/CWP_buildings_l_english.yml:144]: fixed

File: localization/english/CWP/CWP_buildings_l_english.yml (hunk 2 above)
Comment (raxo2222) [reply localization/english/CWP/CWP_buildings_l_english.yml:157]: oops

File: localization/english/CWP/CWP_buildings_l_english.yml (hunk 3 above)
Comment (raxo2222) [reply localization/english/CWP/CWP_buildings_l_english.yml:162]: and done

**Review by DZappala — APPROVED** [review]

**Review by JackEPepper — APPROVED** [review]

**Pull request merged**